SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -140,29 +141,9 @@ Interest InterestFactory::CreatePlayerOwnedActorInterest()
 	return NewInterest;
 }
 
-QueryConstraint InterestFactory::CreateDefinedConstraints()
-{
-	QueryConstraint SystemDefinedConstraints = CreateSystemDefinedConstraints();
-	QueryConstraint UserDefinedConstraints = CreateUserDefinedConstraints();
-
-	QueryConstraint DefinedConstraints;
-
-	if (SystemDefinedConstraints.IsValid())
-	{
-		DefinedConstraints.OrConstraint.Add(SystemDefinedConstraints);
-	}
-
-	if (UserDefinedConstraints.IsValid())
-	{
-		DefinedConstraints.OrConstraint.Add(UserDefinedConstraints);
-	}
-
-	return DefinedConstraints;
-}
-
 QueryConstraint InterestFactory::CreateSystemDefinedConstraints()
 {
-	QueryConstraint CheckoutRadiusConstraint = CreateCheckoutRadiusConstraint();
+	QueryConstraint CheckoutRadiusConstraint = CreateCheckoutRadiusConstraints();
```

Comment:
We should do this once at the very start of the game and cache the constraint. This constraint will always be the same.

Reply:
agreed, will update.